shared/chat/conversation/input-area/normal/typing/index.js

```diff
@@ -64,18 +64,17 @@ type Props = {|
 |}
 
 export const Typing = (props: Props) => (
-  <Kb.Box
-    style={Styles.collapseStyles([
-      styles.isTypingContainer,
-      props.names.size > 0 && styles.isTypingContainerVisible,
-    ])}
-  >
-    <Kb.Box style={styles.typingIconContainer}>
-      <Kb.Animation animationType="typing" containerStyle={styles.isTypingAnimation} />
-    </Kb.Box>
-    <Kb.Text lineClamp={1} type="BodyTiny" style={styles.isTypingText}>
-      <Names names={props.names} />
-    </Kb.Text>
+  <Kb.Box style={styles.isTypingContainer}>
+    {props.names.size > 0 && (
+      <Kb.Box style={styles.typingIconContainer}>
+        <Kb.Animation animationType="typing" containerStyle={styles.isTypingAnimation} />
+      </Kb.Box>
+    )}
+    {props.names.size > 0 && (
+      <Kb.Text lineClamp={1} type="BodyTiny" style={styles.isTypingText}>
+        <Names names={props.names} />
+      </Kb.Text>
+    )}
```

Comment:
Why not just wrap these two with a fragment?

Reply:
that changes the hierarchy which i didn't want to do. aka if this was a box2 and i did that, suddenly the gap would be different. there's only 2 things so it was easy enough to do. if you want to change it later you can